### ISSUE TITLE
[Operator] Failed fuse installation due to unresolved images on OSD 4.2

### ISSUE THREAD

**Issue by JameelB** [syndesisio/syndesis#7111]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description
**NOTE**: This was tested on OSD 4.2

Fuse fails to install successfully via the operator on OpenShift 4.2 for the following reasons:
1. `syndesis-db` deployment config fails to rollout due to unresolved images. Postgresql `9.5` which the `syndesis-db` deployment config uses, is no longer available on OpenShift 4.2. The following versions are available instead:
![image](https://user-images.githubusercontent.com/9078522/68486392-e57fbf00-0238-11ea-9d9c-59a0d6a82adb.png)

2. `fuse-komodo-server` image stream receives the following error when trying to import: 
```
! error: Import failed (InternalError): Internal error occurred: error parsing HTTP 404 response body: invalid character 'F' looking for beginning of value: "File not found.\""
```

This is due to the image stream pointing to `registry.redhat.io/fuse7-tech-preview/data-virtualization-server-rhel7:1.4-11` which is no longer available in the [registry](https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/fuse7-tech-preview/data-virtualization-server-rhel7).

**Comment by JameelB** [comment]
cc @philbrookes

**Comment by lgarciaaco** [comment]
@JameelB which version are you installing and which method are you using? Thanks

**Comment by JameelB** [comment]
Hi @lgarciaaco, we are using the operator version 1.4 `registry.redhat.io/fuse7/fuse-online-operator:1.4`. We are installing Fuse via this operator with the following CR: 
```
apiVersion: "syndesis.io/v1alpha1"
kind: "Syndesis"
metadata:
    name: "integreatly"
spec:
    components:
    db:
      resources: {}
    grafana:
      resources: {}
    meta:
      resources: {}
    prometheus:
      resources: {}
    server:
      features:
        exposeVia3Scale: true
      resources: {}
    upgrade:
      resources: {}
    integration:
      limit: 0
```

**Comment by lgarciaaco** [comment]
@heiko-braun mb we create a ENTESB for this? 

We need to backport https://github.com/syndesisio/syndesis/pull/7255 to 1.7.x and then rebuild the operator. I am not sure what to do with komodo there is no image for 1.4.

**Comment by JameelB** [comment]
@lgarciaaco Closing this issue as both have been resolved in the latest GA version `Fuse 7.5/Operator version 1.5`.
